### ISSUE TITLE
Genesis Informational Adjustments

### ISSUE THREAD

**Issue by KeenanLukeOM** [DXgovernance/infinite-hackathon#24]
Further adjustments to the primary landing section from INTBLUE can be found below. The current [implementation on staging](https://dxgovernance.github.io/infinite-hackathon/#/) can be found on the right, with the proposed changes/initial implementations for reference on the right.

![image](https://user-images.githubusercontent.com/76129769/188248999-4f98929e-ae51-4d2a-ad98-2e6d42161d3e.png)

DETAILS: 
- [ ] TYPOGRAPHY: copy type needs to be bigger 
- [ ] BACKGROUND: Should be sticky in the center of the viewer's screen

Figma for reference: https://www.figma.com/file/Au0PXawZA9FXpgBPI5JMFf/hackersoul-microsite?node-id=381%3A4677

If there are any questions surrounding the above changes, please reach out to myself on Keybase or leave a comment here.

**Comment by KeenanLukeOM** [comment]
The background has moved to the bottom of the page since the last version, this should be centred if possible

![image](https://user-images.githubusercontent.com/76129769/192318678-553d8f1f-0cea-4c21-bafd-65a7150cda91.png)

**Comment by rulfo71** [comment]
you mean so on the background you can see the whole ring, right?

**Comment by KeenanLukeOM** [comment]
Exactly! How it was in the previous build where the orange ring was centred. (If IntBlue has advised otherwise surrounding the placement of the background, ignore this ticket)

![image](https://user-images.githubusercontent.com/76129769/192431081-523166a4-60ae-4489-a8f2-da11e382e71e.png)
